core/src/main/java/com/novoda/merlin/service/ConnectivityChangesForwarder.java

```diff
@@ -0,0 +1,68 @@
+package com.novoda.merlin.service;
+
+import com.novoda.merlin.NetworkStatus;
+import com.novoda.merlin.receiver.ConnectivityChangeEvent;
+import com.novoda.merlin.registerable.bind.BindListener;
+import com.novoda.merlin.registerable.connection.ConnectListener;
+import com.novoda.merlin.registerable.disconnection.DisconnectListener;
+
+class ConnectivityChangesForwarder {
```

Comment:
Has knowledge of the bound listeners of `Merlin` forwarding the relevant information based on a given `ConnectivityChangeEvent`.